app/models/accessible_resource.rb

```diff
@@ -0,0 +1,9 @@
+class AccessibleResource < ApplicationRecord
```

Comment:
👎 This name is super awkward for something that will be used throughout all of controllers.